symengine/eval_arb.h

```diff
@@ -7,8 +7,6 @@
 #ifndef SYMENGINE_EVAL_ARB_H
 #define SYMENGINE_EVAL_ARB_H
 
-#include <symengine/symengine_config.h>
```

Comment:
In this file, you need to include `basic.h` as long as `HAVE_SYMENGINE_ARB` is not defined, so only `symengine_config.h` is enough